website/videoBlog/2022-12-14-Build_Slowly_Changing_Dimensions_Type_2_SCD2_with_Apache_Spark_and_Apache_Hudi_Hands_on_Labs.md

```diff
@@ -8,5 +8,8 @@ image: /assets/images/hudi-video-page-default.png
 navigate: "https://youtu.be/V2LqEvEeU_8"
 tags:
 - guide
+- scd2
+- Slowly Changing Dimensions Type 2
```

Comment:
Is there any convention for the tag names?  Should they all be in lower cases or it depends?

Reply:
Good call. I dint find any right or wrong guidance. But most guidelines suggested being consistent. I can move them all to lowercase and update the ReadMe. Are you okay approving this PR? I can do one cleanup of all tags after landing them all.

Reply:
Yeah, I've already approved the PR.